<!-- -*- mode: XML -*- -->

<launch>
    <node name="dynamixel_left_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: tflex_left_motors
            serial_ports:
                tflex_left_tilt_port:
                    port_name: "/dev/ttyUSB0"
                    baud_rate: 2000000
                    min_motor_id: 1
                    max_motor_id: 10
                    update_rate: 1000
        </rosparam>
    </node >
    <node name="dynamixel_right_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: tflex_right_motors
            serial_ports:
                tflex_right_tilt_port:
                    port_name: "/dev/ttyUSB1"
                    baud_rate: 2000000
                    min_motor_id: 1
                    max_motor_id: 10
                    update_rate: 1000
        </rosparam>
    </node >
    <!-- node name="dynamixel_manager_posterior" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: posterior_motor
            serial_ports:
                posterior_tilt_port:
                    port_name: "/dev/ttyUSB1"
                    baud_rate: 1000000
                    min_motor_id: 2
                    max_motor_id: 2
                    update_rate: 1000
        </rosparam>
    </node -->
</launch>
